src/tooltip/Tooltip.tsx

```diff
@@ -1,357 +1,223 @@
-      xOffset,
-      yOffset,
-      elementWidth,
-      elementHeight,
-      ScreenWidth,
-      ScreenHeight,
-      width,
-      height,
-      withPointer
-    );
-
-    return StyleSheet.flatten([
-      {
-        position: 'absolute',
-        [I18nManager.isRTL ? 'right' : 'left']: x,
-        top: y,
-        width,
-        height,
-        backgroundColor,
-        // default styles
-        display: 'flex',
-        alignItems: 'center',
-        justifyContent: 'center',
-        flex: 1,
-        borderRadius: 10,
-        padding: 10,
-      },
-      containerStyle,
-    ]);
-  };
-
-  renderPointer = (tooltipY: FlexStyle['top']) => {
-    const { yOffset, xOffset, elementHeight, elementWidth } = this.state;
-    const { backgroundColor, pointerColor } = this.props;
-    const pastMiddleLine = yOffset > (tooltipY || 0);
-
-    return (
-      <View
-        style={{
-          position: 'absolute',
-          top: pastMiddleLine ? yOffset - 13 : yOffset + elementHeight - 2,
-          [I18nManager.isRTL ? 'right' : 'left']:
-            xOffset +
-            getElementVisibleWidth(elementWidth, xOffset, ScreenWidth) / 2 -
-            7.5,
-        }}
-      >
-        <Triangle
-          style={{ borderBottomColor: pointerColor || backgroundColor }}
-          isDown={pastMiddleLine}
-        />
-      </View>
-    );
-  };
-
-  getTooltipHighlightedButtonStyle = (): ViewStyle => {
-    const { highlightColor } = this.props;
-    const { yOffset, xOffset, elementWidth, elementHeight } = this.state;
-
-    return {
-      position: 'absolute',
-      top: yOffset,
-      [I18nManager.isRTL ? 'right' : 'left']: xOffset,
-      backgroundColor: highlightColor,
-      overflow: 'visible',
-      width: elementWidth,
-      height: elementHeight,
-    };
-  };
-
-  renderTouchableHighlightedButton = () => {
-    const TooltipHighlightedButtonStyle = this.getTooltipHighlightedButtonStyle();
-
-    return (
-      <TouchableOpacity
-        testID="tooltipTouchableHighlightedButton"
-        onPress={() => this.toggleTooltip()}
-        style={TooltipHighlightedButtonStyle}
-      >
-        {this.props.children}
-      </TouchableOpacity>
-    );
-  };
-
-  renderStaticHighlightedButton = () => {
-    const TooltipHighlightedButtonStyle = this.getTooltipHighlightedButtonStyle();
-
-    return (
-      <View style={TooltipHighlightedButtonStyle}>{this.props.children}</View>
-    );
-  };
-
-  renderHighlightedButton = () => {
-    const { closeOnlyOnBackdropPress } = this.props;
-    if (closeOnlyOnBackdropPress) {
-      return this.renderTouchableHighlightedButton();
+const ToolTip2: React.FunctionComponent<ToolTip2Props> = ({
+  children,
+  title,
+  popover,
+  withPointer = true,
+  overlayColor,
+  backgroundColor = '#616161',
+  pointerColor,
+  toggleAction = 'onPress',
+  onShow = () => {},
+  onDismiss = () => {},
+  visible = false,
+  width: propWidth = 200,
+  closeOnlyOnBackdropPress = false,
```

Comment:
Is it possible to avoid the backdrop at all. Usually, tooltip shouldn't block any interaction with other components.